### PR TITLE
EWPP-2016: Improve display of person teaser.

### PR DESCRIPTION
**Pull request by imanoleguskiza**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: modules/oe_theme_content_person/oe_theme_content_person.module [hunk 1]
```diff
@@ -11,6 +11,20 @@ use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
 use Drupal\Core\Entity\EntityInterface;
 use Drupal\oe_theme\ValueObject\ImageValueObject;
 
+/**
+ * Implements hook_theme().
+ */
+function oe_theme_content_person_theme() {
```
Comment (sergepavle) [inline modules/oe_theme_content_person/oe_theme_content_person.module]:
```suggestion
function oe_theme_content_person_theme(): array {
```

**Review by sergepavle — DISMISSED** [review]

File: templates/field/field--node--oe-departments--oe-person--teaser.html.twig [hunk 2]
```diff
@@ -6,4 +6,4 @@
  * @see ./core/themes/stable/templates/field/field.html.twig
  */
 #}
-{%- include 'field--multiple-values--pipe-separated.html.twig' -%}
+{%- include 'field--multiple-values--comma-separated.html.twig' -%}
```
Comment (22Alexandra) [inline templates/field/field--node--oe-departments--oe-person--teaser.html.twig:9]: This change has to be done for all the departments fields according to the ticket.

**Review by 22Alexandra — CHANGES_REQUESTED** [review]
This request from the ticket is missing: `Point 1. 2. and 3. should apply to all instances of the content item pattern, not only on person teaser.`

So we need to hide the contact name everywhere we render contacts and separate departments by comma instead of pipe too.

**Review by 22Alexandra — DISMISSED** [review]

File: composer.json [hunk 3]
```diff
@@ -58,7 +58,8 @@
         "openeuropa/rdf_skos": "^1.0",
         "openeuropa/task-runner-drupal-project-symlink": "^1.0-beta5",
         "phpspec/prophecy-phpunit": "^2",
-        "symfony/dom-crawler": "^4.4.12"
+        "symfony/dom-crawler": "^4.4.12",
+        "phpmd/phpmd": "2.11.1"
```
Comment (yenyasinn) [inline composer.json]: Do we still need it? If not, can be removed before merge.

**Review by yenyasinn — DISMISSED** [review]

**Review by yenyasinn — DISMISSED** [review]

**Review by yenyasinn — APPROVED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**